habana/1.13.0/ubi8-python-3.8/Pipfile

```diff
@@ -0,0 +1,61 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+# Habana requirements
+habana_media_loader = "==1.13.0.463"
+
+# TensorFlow and useful extensions
+habana-tensorflow = "==1.13.0.463"
+tensorflow-cpu = "==2.12.1"
+tensorflow-io = "==0.32.0"
+tensorboard = "~=2.12.2"
+tf2onnx = "~= 1.13.0"
+
+# Pytorch and useful extension
+habana-pyhlml = "==1.13.0.463"
+
+# Below pkgs are required for pytorch-fork
+cffi = "==1.15.1"
+typing-extensions = ">=4.5.0"
+numpy = "==1.23.5"
+ninja = ">=1.11.1"
+pyyaml = "==6.0"
+pybind11 = "==2.10.4"
+mkl = "==2023.1"
+mkl-include = "==2023.1"
+
+# Parent image requirements to maintain cohesion
```

Comment:
The packages from line 33 to 58 have been updated to newer versions, could you take a look and update  this file as well? Ref link: https://github.com/opendatahub-io/notebooks/blob/main/jupyter/datascience/ubi8-python-3.8/Pipfile#L9-L45